seed.js

```diff
@@ -1,57 +1,235 @@
+    'testing@fsa.com'
+  );
+  Omri.isAdmin = true;
+  instructors.push( Omri );
+  let creatingInstructors = instructors.map( function ( instructorObj ) {
+    return User.create( instructorObj );
+  } );
+  return Promise.all( creatingInstructors );
+};
 
-    var creatingUsers = users.map(function (userObj) {
-        return User.create(userObj);
-    });
+/** seeds up to n organizations */
+const seedOrganizations = function ( n = Math.ceil( Math.random() * 100 ) ) {
+  let organizations = User.findAll( {
+      limit: n
+    } )
+    .map( user => {
+      let name = faker.company.companyName();
+      return user.createOrganization( {
+        name
+      }, {
+        role: 'creator'
+      } )
+    } )
+  return Promise.all( organizations );
+}
 
-    return Promise.all(creatingUsers);
+const seedTeams = function ( n = Math.ceil( Math.random() * 100 ) ) {
+  let teams = Organization.findAll( {
+      include: [ {
+        model: User
+      } ]
+    } )
+    .map( organization => [
+      organization,
+      organization.users[ 0 ],
+      faker.helpers.shuffle( faker.commerce.productName()
+        .split( ' ' ) )
+      .join( '' )
+    ] )
+    .map( result => {
+      let [ organization, user, name ] = result;
+      return user.createTeam( {
+          name,
+          creatorId: user.id
+        }, {
+          role: 'instructor',
+        } )
+        .then( team => team.setOrganization( organization ) )
+    } )
+  return Promise.all( teams );
+}
 
-};
+let seedAssessments = function () {
+  let assessments = User.findAll( {
+      include: [ Team ]
+    } )
+    .map( instructor => {
+      if ( !instructor.teams[ 0 ] ) return
+      let name = faker.lorem.words( Math.ceil( Math.random() * 20 ) );
+      let description = faker.lorem.paragraph();
+      let tags = faker.random.words( Math.ceil( Math.random() * 10 ) )
+        .toLowerCase()
+        .split( ' ' );
+      let repoUrl = faker.internet.url();
+      let instructorId = instructor.id;
+      let team = instructor.teams[ 0 ].userTeam;
+      let teamId = team.id;
+      return Assessment.create( {
+        name,
+        description,
+        tags,
+        repoUrl,
+        instructorId,
+        teamId
+      } )
+    } )
+  return Promise.all( assessments );
+}
+
+let seedQuestions = function ( n = 20 ) {
+  let questions = Assessment.findAll()
+    .map( assessment => {
+      let N = Math.ceil( Math.random() * n );
+      let questions = Array.from( {
+        length: N
+      }, question => {
+        let prompt = faker.lorem.words( Math.ceil( Math.random() * 20 ) );
+        let answer = faker.lorem.words( Math.ceil( Math.random() * 20 ) );
+        let assessmentId = assessment.id;
+        return {
+          prompt,
+          answer,
+          assessmentId
+        };
+      } )
+      return Question.bulkCreate( questions );
+    } )
+  return Promise.all( questions );
+}
+
+let seedRubrics = function ( n = 20 ) {
+  console.log( 'seeding rubrics' );
```

Comment:
Do you not want them in each stage then? Why just rubrics and like one other if I remember correctly